sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Bugs Fixed
 - `AzurePowerShellCredential` now handles the case where it falls back to legacy PowerShell without relying on the error message string.
 
+### Other Changes
+- `DefaultAzureCredential` now sends a probe request with no retries for IMDS managed identity environments with no retries to avoid excessive retry delays when the IMDS endpoint is not available. This should improve credential chain resolution for dev-time scenarios.
```

Comment:
Wording is odd here since you mention "with no retries" twice in the same sentence. I'd also expand - for translatability - "dev-time" to "developer-time".

Reply:
good catch - This was a bad edit.